Added case study section & remove link to The Hive. (#34)
ServiceInnovationLab/bagel-box: pages/feedback-loop.md

  <a name="Limitations"></a>
  <h3>Limitations</h3>

  <p>The participants you engage with when designing your feedback may not represent everyone in their community, so acknowledging and recognising this is important. Also due to government process and time pressures placed on consultations, this process is not always possible. If this is the case try and think about what is possible, what is the purpose of your feedback and who you want to access the feedback.</p>

  <a name="Case-Study"></a>
  <h3>Case Study</h3>

  <p>We have tried to explore and follow this process with The Hive. The Hive is a Ministry of Youth Development (MYD) Youth Voice Project that uses an innovative approach to increasing young people’s participation in the government policy development process. MYD worked in partnership with design agency Curative and 15 rangatahi on this project.</p>

>   <p>The 15 rangatahi (known as the Hivers) were invited to participate in a co-design hui to identify and create solutions for "How might we enable a two-way communication platform between young people and policy makers?" They were presented with the issue, and went on to design The Hive which included youth centric website, with accessible and relatable content, and a social media drive.</p>

Reviewer: ```suggestion
  <p>The 15 rangatahi (known as the Hivers) were invited to participate in a co-design hui to identify and create solutions for "How might we enable a two-way communication platform between young people and policy makers?" They were presented with the issue, and went on to design The Hive - a youth centric website with accessible and relatable content - and a social media drive.</p>
```

>   <p>Following a successful consultation, using the Biodiversity, The Hivers were consulted on how the results should be released. The methods used, social media, video, info graphics, reflects the consultation methods and the community we wanted to engage with.</p>

Reviewer: ```suggestion
  <p>Following a successful consultation, on the Biodiversity Strategy, The Hivers were consulted on how the results should be released. The methods used - social media, video, and infographics - reflects the consultation methods and the community we wanted to engage with.</p>
```

>   <p>Through the process The Hivers designed, created and were consulted throughout. Resulting in the entire process been relevant and accessible for the people we are wanted to engage.

Reviewer: ```suggestion
  <p>Through the process The Hivers designed, created and were consulted throughout. As a result, the entire process was relevant and accessible for the people we wanted to engage.
```
